content/docs/next-release/operator.md

```diff
@@ -0,0 +1,668 @@
+
+<10> Define annotations to be applied to all deployments (not services). These can be overridden by annotations defined on the individual components.
+
+You can view example custom resources for different Jaeger configurations [on GitHub](https://github.com/jaegertracing/jaeger-operator/tree/master/deploy/examples).
+
+# Configuring the Custom Resource
+
+<!--TODO
+esIndexCleaner
+Spark dependencies
+-->
+
+You can use the simplest example (shown above) and create a Jaeger instance using the defaults, or you can create your own custom resource file.
+
+## Storage options
+
+### Cassandra storage
+
+When the storage type is set to Cassandra, the operator will automatically create a batch job that creates the required schema for Jaeger to run. This batch job will block the Jaeger installation, so that it starts only after the schema is successfuly created. The creation of this batch job can be disabled by setting the `enabled` property to `false`:
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: cassandra-without-create-schema
+spec:
+  strategy: allInOne
+  storage:
+    type: cassandra
+    cassandraCreateSchema:
+      enabled: false # <1>
+```
+<1> Defaults to `true`
+
+Further aspects of the batch job can be configured as well. An example with all the possible options is shown below:
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: cassandra-with-create-schema
+spec:
+  strategy: allInOne # <1>
+  storage:
+    type: cassandra
+    options: # <2>
+      cassandra:
+        servers: cassandra
+        keyspace: jaeger_v1_datacenter3
+    cassandraCreateSchema: # <3>
+      datacenter: "datacenter3"
+      mode: "test"
+```
+<1> The same works for `production` and `streaming`.
+
+<2> These options are for the regular Jaeger components, like `collector` and `query`.
+
+<3> The options for the `create-schema` job.
+
+{{< info >}}
+The default create-schema job uses `MODE=prod`, which implies a replication factor of `2`, using `NetworkTopologyStrategy` as the class, effectively meaning that at least 3 nodes are required in the Cassandra cluster. If a `SimpleStrategy` is desired, set the mode to `test`, which then sets the replication factor of `1`. Refer to the [create-schema script](https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/cassandra/schema/create.sh) for more details.
+{{< /info >}}
+
+### Elasticesearch storage
+
+Under some circumstances, the Jaeger Operator can make use of the [Elasticsearch Operator](https://github.com/openshift/elasticsearch-operator) to provision a suitable Elasticsearch cluster.
+
+{{< danger >}}
+This feature is experimental and has been tested only on OpenShift clusters. Elasticsearch also requires the memory setting to be configured like `minishift ssh -- 'sudo sysctl -w vm.max_map_count=262144'`. Spark dependencies are not supported with this feature [Issue #294](https://github.com/jaegertracing/jaeger-operator/issues/294).
```

Comment:
@jpkrohling  Was thinking of removing the "experimental" part - but leave only tested on openshift? and change to warning.

Reply:
By the way, `minishift` is no more, but not sure what can be used instead...

Reply:
removed the minishift part as well for now.